posthog/models/cohort/cohort.py

```diff
@@ -181,6 +182,20 @@ class Cohort(FileSystemSyncMixin, RootTeamMixin, models.Model):
         help_text="Type of cohort based on filter complexity",
     )
 
+    # Realtime evaluation support flag
+    realtime_supported = models.BooleanField(
```

Comment:
This feels weird - surely this is what the type indicates - we don't need two values here.